proto/planetmintgo/dao/query.proto

```diff
@@ -26,19 +26,19 @@ service Query {
   
   // Queries a list of GetMintRequestsByHash items.
   rpc GetMintRequestsByHash (QueryGetMintRequestsByHashRequest) returns (QueryGetMintRequestsByHashResponse) {
-    option (google.api.http).get = "/planetmint/dao/get_mint_requests_by_hash/{hash}";
+    option (google.api.http).get = "/planetmint/dao/mint_requests_hash/{hash}";
   
   }
   
   // Queries a list of MintRequestsByAddress items.
   rpc MintRequestsByAddress (QueryMintRequestsByAddressRequest) returns (QueryMintRequestsByAddressResponse) {
-    option (google.api.http).get = "/planetmint/dao/mint_requests_by_address/{address}";
+    option (google.api.http).get = "/planetmint/dao/mint_requests_address/{address}";
```

Comment:
shouldn't we name this one to
planetmint/dao/ mint_requests/address/{address} like the others below?

Reply:
you're completely right. however the corresponding cli command needs then turned from `planetmint-god query dao mint-requests-address` into `planetmint-god query dao mint-requests address`. Meaning `address` needs to be a sub-commmand of mint-requests....

I'll have a look how to do that...